src/test/java/org/sagebionetworks/bridge/exporter/integration/ExportTest.java

```diff
@@ -0,0 +1,1142 @@
+                        break;
+                    }
+                    case "AAA": {
+                        assertEquals("Yes", columnValue);
+                        break;
+                    }
+                    case "BBB.fencing": {
+                        assertEquals("true", columnValue);
+                        break;
+                    }
+                    case "BBB.football": {
+                        assertEquals("false", columnValue);
+                        break;
+                    }
+                    case "BBB.running": {
+                        assertEquals("true", columnValue);
+                        break;
+                    }
+                    case "BBB.swimming": {
+                        assertEquals("false", columnValue);
+                        break;
+                    }
+                    case "BBB.3": {
+                        assertEquals("true", columnValue);
+                        break;
+                    }
+                    default: fail("not expected column type: " + headerName + ": " + columnValue);
+                }
+            }
+        }
+    }
+
+    private void verifyExportTime(long endDateTimeEpoch, boolean ignoreLastExportDateTime, boolean exportAll)
+            throws InterruptedException {
+        if (!ignoreLastExportDateTime) {
+            if (!exportAll) {
+                Item lastExportDateTime = ddbExportTimeTable.getItem("studyId", studyId);
+                long lastExportDateTimeEpoch = lastExportDateTime.getLong("lastExportDateTime");
+                assertNotNull(lastExportDateTimeEpoch);
+
+                // the time recorded in export time table should be equal to the date time we submit to the export request
+                assertEquals(endDateTimeEpoch, lastExportDateTimeEpoch);
+            } else {
+                Iterable<Item> scanOutcomes;
+                long lastExportDateTimeEpoch;
+                for (int i = 0; i < 3; i++) {
+                    scanOutcomes = ddbScanHelper.scan(ddbExportTimeTable);
+
+                    // verify if all studies' last export date time are modified
+                    for (Item item: scanOutcomes) {
+                        lastExportDateTimeEpoch = item.getLong("lastExportDateTime");
+                        if (endDateTimeEpoch != lastExportDateTimeEpoch) {
+                            LOG.info("Re-try get last export date time: " + i);
+                            TimeUnit.SECONDS.sleep(10);
+                        }
+                    }
+                }
+
+                scanOutcomes = ddbScanHelper.scan(ddbExportTimeTable);
+
+                // verify if all studies' last export date time are modified
+                for (Item item: scanOutcomes) {
+                    lastExportDateTimeEpoch = item.getLong("lastExportDateTime");
+                    assertEquals(endDateTimeEpoch, lastExportDateTimeEpoch);
+                }
```

Comment:
We *just* scanned the DDB table and verified that everything was equal. Why do we need to scan it again?